fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -1909,4 +2009,21 @@ public TTransactionStatus getTxnStatus(long txnId) {
                 return TTransactionStatus.UNKNOWN;
         }
     }
+
+    private void writeTransactionStateEditLog(TransactionState transactionState) {
+        if (Config.use_lock_manager) {
+            if (transactionState.getTransactionStatus() != TransactionStatus.PREPARE
+                    || transactionState.getSourceType() == TransactionState.LoadJobSourceType.FRONTEND) {
+                // if this is a prepared txn, and load source type is not FRONTEND
+                // no need to persist it. if prepare txn lost, the following commit will just be failed.
+                // user only need to retry this txn.
+                // The FRONTEND type txn is committed and running asynchronously, so we have to persist it.
+                long start = System.currentTimeMillis();
+                editLog.logInsertTransactionState(transactionState);
+                LOG.debug("insert txn state for txn {}, current state: {}, cost: {}ms",
+                        transactionState.getTransactionId(), transactionState.getTransactionStatus(),
+                        System.currentTimeMillis() - start);
+            }
+        }
+    }
```

Comment:
```suggestion
    private void persistTxnStateInTxnLevelLock(TransactionState transactionState) {
        if (Config.load_using_fine_granularity_lock_enabled) {
            doWriteTxnStateEditLog();
        }
    }
```

Reply:
The function name is not illustrative, suggest `persistTxnStateInTxnLevelLock()`.
Besides, i found that `Config.use_lock_manager` can also be quite confusing, because here we want to use fine granularity lock mechanism to optimize the loading process, `lock manager` is just the foundamental tool we are using to achieve that, and even `Config.use_lock_manager` is `false`, we can still use it in other cases which are not related to loading process.